server/api/studio/internal/service/file.go

```diff
@@ -88,36 +114,58 @@ func (f *fileService) FilesIndex() (data *types.FilesIndexData, err error) {
 		if fileInfo.IsDir() {
 			continue
 		}
+		var fileConfig db.File
+		result := db.CtxDB.Where("name = ?", fileInfo.Name()).First(&fileConfig)
+		if result.Error != nil {
+			logx.Errorf("get file config record in db error %v", result.Error)
+			fileConfig.Delimiter = ","
+			fileConfig.WithHeader = false
+		}
```

Comment:
Why init with `","`, `false`?

Reply:
Considering that the user can directly upload the file to the corresponding directory of the server, the configuration record of the file may not exist in the database, so the default value is set and returned to the front end.